content/en/os/1.15.x/concepts/host-containers/_index.markdown

```diff
@@ -0,0 +1,86 @@
+The control container provides a pathway to connect to and interact with the host as well as conveniently access the Bottlerocket API.
+The control container also provides an entry point to the admin container.
+The admin container allows you to more deeply interact with the host as it mounts the root filesystem and has elevated privileges.
+
+The control and admin containers are not actually special: both are just host containers with specific privileges.
+Case in point, you can disable these containers as-needed and you can even fully replace the functionality of both the host and admins containers with your own host containers.
+
+## Properties of host containers
+
+Host containers have a number of different properties from orchestrated containers.
+
+### Runtime
+
+Bottlerocket has two parallel instances of `containerd`: one for your orchestrated workloads and one for host containers.
+
+### API Access
+
+All host containers automatically mount the API socket (`/run/api.sock`) and the API Client (`/usr/local/bin/apiclient`).
+This allows you to run API commands and manipulate Bottlerocket settings from within the host containers without explicitly mounting these resources.
+
+### Lifecycle & restarts
+
+Host containers start and stop based on the value of their {{< setting-reference setting="settings.host-containers.<container>.enabled" >}}enabled{{</ setting-reference >}} setting.
+If a host container exits, Bottlerocket automatically restarts the container after 45 seconds.
+
+### Superpower
+
+Host containers can have high levels of privilege.
+See {{< setting-reference setting="settings.host-containers.<container>.superpowered" >}}{{</ setting-reference >}} for more details.
+
+### Updates
+
+Host containers do not update automatically.
+Updates only occur when you update the {{< setting-reference setting="settings.host-containers.<container>.source" >}}{{</ setting-reference >}}.
+
+### User data
+
+Distinct, but inspired by instance user data, host containers pass arbitrary[^1] base64-encoded data from the API into a file[^2].
+
+### Persistent storage
+
+All host containers have persistent storage that survive node reboots as well as container stop/start cycles[^2].
+
+## Use Cases
+
+When considering if you should use a host container, evaluate the specific properties of host containers and if you need to manage the lifecycle with your orchestrator.
+Generally, you should try to solve your problem with an orchestrated container and only turn to a host container for low-level use-cases.
+
+Examples:
+
+- The control container is a host container because it needs to be available early to give you access to the OS.
```

Comment:
I think the control container is a host container because it allows users to manage their hosts with SSM. e.g. in a running cluster, if they want to update their settings, they can send an SSM command an update the entire cluster without interacting with the host.

Reply:
That's interesting, but that could be done with a non-host container too, right?

This line is basically a re-wording of this [from the current README](https://github.com/bottlerocket-os/bottlerocket/blob/develop/README.md?plain=1#L1323): 

```
We use them for the control container because it needs to be available early to give you access to the OS
```

Reply:
It could, but then you need to configure the orchestrated container to have access to the API. 

I think what we need is to clarify that "early" means "before" the orchestrated containers. And, that this is not the only use case for the control container. My understanding is that we want the SSM agent to both have access to the host for debugging purposes, and execute commands through SSM to interact with the host's API. Say you have a new certificate, but you don't want to apply it through a bootstrap container because that requires to reboot the instance. The control container allows you to send SSM commands to all the running hosts to update the API and trust the new certificate. Or if the user data is too big, you can send an SSM command that configures a new host container that pulls down the certificates' bundle from S3 and applies it to the API.

Reply:
I can definitely clarify the temporal reason for the orchestrated containers.

I'm hesitant to include anything about SSM and control containers because this may not be a given in the future across all variants. 

For example, outside of AWS, SSM can be very expensive and jumping directly to a admin container with SSH running would be an un-need escalation of privileges. So, it's not out of the realm of possibility for some control container w/o SSM to exist in the future.